src/main/java/core/basesyntax/dao/BetDao.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Bet;
+import java.util.List;
+
+public interface BetDao {
+    void add(Bet bet);
+
+    List<Bet> getAll();
+
+    boolean remove(Bet bet);
+
+    void updateMoney(Bet bet, int money);
```

Comment:
let's remove this method

if our bets had some id, this functionality would make sense

but now we can have multiple different bets with identical `money` and `chanceOfVictory`
you are always updating the first one